### DISCUSSION TITLE
Chrome Web Store Lock

### DISCUSSION THREAD

**Discussion by kalenlechner** [dragon731012/-WORKING-bookmarklets-and-games#44] [Q&A]
If you go to school with a Chromebook, you probably can't get extensions, themes, etc. because of a lock. 

![Screenshot 2024-05-17 08 27 57](https://github.com/dragon731012/-WORKING-bookmarklets-and-games/assets/169999696/22b79d80-7c87-4949-bec6-571c61892516)

Is it possible to workaround this? May definitely be a stretch, but it would help to even know if it is possible.

**Comment by Wheels522** [comment]
Oh I know! Factory reset ur Chromebook is one

**Comment by Wheels522** [comment]
Or u could use ltbeef

**Reply by kalenlechner** [reply]
If you don't mind me asking, what is Itbeef?
